lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
If I'm reading this PR correctly, the certificate generation process won't know the difference between a student who is in a POC or not, and POCs can have different grading policies from each other and the original course that they're derived from. The certificate itself will carry the name of the original course. So two people who have identical scores for every problem in the course could have different grades (and therefore one could get a certificate and the other not) depending on the POC grading policy. Is that ok?